examples/docs_snippets/docs_snippets/integrations/bigquery/tutorial/resource/downstream.py

```diff
@@ -0,0 +1,22 @@
+from dagster_gcp import BigQueryResource
+from google.cloud import bigquery as bq
+
+# start_example
+from dagster import asset
+
+from .create_table import iris_data
+
+# this example uses the iris_dataset asset from Step 2
+
+
+@asset(deps=[iris_data])
+def iris_setosa(bigquery: BigQueryResource) -> None:
+    job_config = bq.QueryJobConfig(destination="iris.iris_setosa")
```

Comment:
I have peasant-tier GCP SDK skillz, but does this set the destination as `iris.iris_setosa` and then uses the SQL query to populate the DDL?

I'm wondering how familiar BQ users are with this or if it would be more straightforward to not set the `job_config` and instead have the SQL query be something like:


```sql
create or replace table `iris`.`iris_setosa` as
select
    *
from `iris`.`iris_data`
where species = 'Iris-setosa'
```

Thoughts/reasonsing?

Reply:
this is the pattern the BQ docs tell you to do, so i'd assume people are familiar with it? https://cloud.google.com/bigquery/docs/tables 

tbh i don't know BQ well either, so i'm going off my best guess based on docs

Reply:
cool ignore me then!